data-service/src/main/java/uk/gov/gchq/palisade/service/data/stream/PropertiesConfigurer.java

```diff
@@ -53,7 +53,9 @@
 /**
  * Parse and convert Spring maps and lists to Akka configs.
  */
+@SuppressWarnings("java:S5998")
 public class PropertiesConfigurer extends PropertySourcesPlaceholderConfigurer implements InitializingBean {
+
     private static final Pattern INDEXED_PROPERTY_PATTERN = Pattern.compile("^\\s*(?<path>\\w+(?:\\.\\w+)*)\\[(?<index>\\d+)\\]\\.*(.*?)$");
     private static final int PROPERTY_PATH = 1;
     private static final int PROPERTY_INDEX = 2;
```

Comment:
One more small thing, we have these labels on the regex groups but don't use them...
```suggestion
    private static final Pattern INDEXED_PROPERTY_PATTERN = Pattern.compile("^\\s*(?<path>\\w+(?:\\.\\w+)*)\\[(?<index>\\d+)\\]\\.*(?<tail>.*?)$");
    private static final String PROPERTY_PATH = "path";
    private static final String PROPERTY_INDEX = "index";
    private static final String PROPERTY_TAIL = "tail";
```

Reply:
That change request should really cover line 62, but github wouldn't let me...